### DISCUSSION TITLE
求助：Mac M1应该如何下载和安装ZLibrary？

### DISCUSSION THREAD

**Discussion by anneeeee** [tw93/Pake#234] [Q&A] [answered]
如题，下载了Release中的三个，其中.APPIMG 和.deb打不开，另一个安装后显示：无法打开“ZLibrary”，因为无法验证开发者。

**Comment by tw93** [comment, marked as answer]
mac 使用mac版本即可，无法验证开发者，参考这个即可 https://blog.csdn.net/qq_32828933/article/details/108334190

**Reply by anneeeee** [reply]
我的电脑只有两个选项：移到废纸篓和取消..
<img width="260" alt="image" src="https://user-images.githubusercontent.com/16276686/211195961-c2f9cd13-d3a9-4ae3-b5e0-60ef4552b15c.png">

**Reply by Tlntin** [reply]
> 我的电脑只有两个选项：移到废纸篓和取消.. <img alt="image" width="260" src="https://user-images.githubusercontent.com/16276686/211195961-c2f9cd13-d3a9-4ae3-b5e0-60ef4552b15c.png">

https://www.cnblogs.com/Formulate0303/p/15476757.html

**Reply by hzyhbk** [reply]
去访达里面找到app，右键选择打开即可

**Reply by anneeeee** [reply]
> 去访达里面找到app，右键选择打开即可

感谢，可以打开了，但为什么打开是空白的呢~~
<img width="1200" alt="image" src="https://user-images.githubusercontent.com/16276686/215374077-99cedcda-94fa-4509-a583-db6c25bf7141.png">

**Reply by Tlntin** [reply]
试试新的。

**Reply by anneeeee** [reply]
> 试试新的。

可以了，感谢

**Comment by S89891515** [comment]
<img width="782" alt="截屏2024-05-31 17 18 26" src="https://github.com/tw93/Pake/assets/171265877/607582ad-5bcd-443d-8965-875320e5d5f3">
